run_scripts/Readme.md

```diff
@@ -0,0 +1,40 @@
+## Step 1. Create the Loader and execute scripts. 
+First create 2 text files for clients and yugabyte nodes named `clients.txt` and `yb_nodes.txt`.
+
+## Step 2. Configure the client nodes.
+This can be done as follows. Make sure that the environment has the ssh user
+exported to the variable `SSH_USER` and the additional SSH AND SCP arguments
+like the pem file or the port exported as `SSH_ARGS` and `SCP_ARGS`.
+```sh
+./setup_clients.sh
+```
+
+## Step 3. Create the TPCC tables.
+This can be done from by:
+```sh
+g++ --std=c++11 run_tpcc_on_clients.cpp && ./a.out create
```

Comment:
we don't need to compile each time right?
Also for clarity, we can give the output fille a better name like `run_tpcc_on_clients` instead of the default `a.out`

Reply:
Done.